cypress/utils/dashboards/data_explorer/elements.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const DATA_EXPLORER_PAGE_ELEMENTS = {
```

Comment:
if we don't use OOP any more, should we still wrap constants in a big object? we don't need to create constants for test-subjs. The benefit for constants here is that if the test-id needs to change, we only update it in one place. But we rarely update test-subjs. The existing tests and our legacy tests all just call test-subjs directly.